duckduckgo/chrome-zeroclickinfo
# Start modularizing the styles

**Reviewer:** @laurengarcia 
**CC:** @jdorweiler 



## Description:

Start modularizing the popup styles: separate files for different popup elements (vars, buttons, etc) which are then included into a main.scss file.

That file is compiled by the sass Grunt task, into public/css/popup.css

## Steps to test this PR:
Coming soon
1. Run `grunt`
2. Load extension in Chrome
3. Open the popup
4. Make sure the styles look exactly the same as before; navigate to some webpages, hover - everything should look exactly the same

## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
